Userland/Libraries/LibJS/Parser.cpp

```diff
@@ -3050,6 +3076,55 @@ NonnullRefPtr<VariableDeclaration> Parser::parse_variable_declaration(bool for_l
     return declaration;
 }
 
+NonnullRefPtr<UsingDeclaration> Parser::parse_using_declaration(JS::Parser::IsForLoopVariableDeclaration is_for_loop_variable_declaration)
```

Comment:
We're already in the parser here, no need to specify `JS::Parser` (also in two other places)

Reply:
or maybe `Parser::` is needed? But definitely not `JS::` :)

Reply:
That was just me trusting the CLion auto complete without checking